src/peers/AddConnection/AddConnection.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { translate } from 'react-i18next'
+import { connect } from 'redux-bundler-react'
+import toUri from 'multiaddr-to-uri'
+
+import Icon from '../../icons/StrokeDecentralization'
+import Button from '../../components/button/Button'
+import Overlay from '../../components/overlay/Overlay'
+import TextInputModal from '../../components/text-input-modal/TextInputModal'
+
+class AddConnection extends React.Component {
+  state = {
+    open: false
+  }
+
+  toggleModal = () => {
+    this.setState({
+      open: !this.state.open
+    })
+  }
+
+  addConnection = (maddr) => {
+    this.props.doConnectSwarm(maddr)
+    this.toggleModal()
+  }
+
+  validate = (maddr) => {
```

Comment:
@hacdias can we switch this to `peerMultiaddr` check from  [is-ipfs v0.6.0](https://github.com/ipfs/is-ipfs/releases/tag/v0.6.0)? 
It gives better validation (confirms the multiaddr represents IPFS peer). 

> ### `isIPFS.peerMultiaddr(addr)`
> 
> Returns `true` if the provided `string`, [`Multiaddr`](https://github.com/multiformats/js-multiaddr) or `Buffer` represents a valid "IPFS Peer" multiaddr (matching [`IPFS` format from `mafmt`](https://github.com/multiformats/js-mafmt#api)) or `false` otherwise.

Reply:
Done!!! 😄